sources/packages/backend/libs/test-utils/src/factories/sfas-application.ts

```diff
@@ -0,0 +1,46 @@
+import { SFASApplication } from "@sims/sims-db";
+import { getISODateOnlyString } from "@sims/utilities";
+import { DataSource } from "typeorm";
+import * as faker from "faker";
+
+/**
+ * Create and save fake SFAS application.
+ * @param dataSource data source to persist the SFAS application.
+ * @param options SFAS application options.
+ * - `initialValues` SFAS application initial values.
+ * @returns persisted SFAS application.
+ */
+export async function saveFakeSFASApplication(
```

Comment:
As per the standards that we are following, the primary responsibility of the factory is to create fake object, not persisting it. 
 
`createFakeSFASApplication` is what we create to return a fake object. like this

![image](https://github.com/bcgov/SIMS/assets/54600590/bb3ce56e-335e-474a-a45e-7aecca482b48)